README.md

```diff
@@ -1,5 +1,28 @@
 # T-asapp
 Proyecto ingeniería software
 
-PROYECTO DE GESTIÓN LOGÍSTICA DE ENVÍO DE PAQUETERÍA
+PROYECTO DE GESTIÓN LOGÍSTICA DE ENVÍO DE PAQUETERÍA A CENTROS EDUCATIVOS DE ANDALUCÍA
+
+Este proyecto pertenece a la materia de Ingeniería del Software del Master en Programación y Diseño de Aplicaciones
+de EDIAE y T-Systems.
+
+El proyecto parte de la necesidad de gestionar un sistema de paquetería orientado a facilita a empresas de paquetería
+el envío de diferntes tipos de productos y paquetesa centros educativos de Andalucía. 
+
+El cliente necesita poder gestionar dichos envíos envíos, bien de documentación o material teniendo 
+en cuenta en número de personas de la plantilla de los centros y se tendrá en cuenta 
```

Comment:
Estáis explicando el contexto del problema, pero para empezar no está claro cuál es el problema, y para seguir tampoco se acaba de entender qué tiene que ver el envío de paquetería con la plantilla.